sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -302,6 +308,12 @@ public WriteTables(
     this.kmsKey = kmsKey;
   }
 
+  public WriteTables<DestinationT> withSchemaUpdateOptions(
+      Set<SchemaUpdateOption> schemaUpdateOptions) {
+    this.schemaUpdateOptions = schemaUpdateOptions;
+    return this;
+  }
```

Comment:
Gladly, `WriteTables` is not public, so I'd think it's better to change the constructor, and make the attribute final.

Reply:
Right on. I'll update this to that mechanism instead then. I haven't peeked at integration tests yet, but that sounds reasonable to me too.

Reply:
I pushed a mini update to address this bit. Going to see about an integration test when I next get a chance to look at this.